### PR TITLE
Update README, add Gradle, Scala

### PR DESCRIPTION
**Pull request by Malax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ The builder images use Heroku's [stack images](https://github.com/heroku/stack-i
 | [`heroku/builder:22`][builder-tags]                 | [`heroku/heroku:22-cnb-build`][heroku-tags] | Recommended |
 | [`heroku/builder-classic:22`][builder-classic-tags] | [`heroku/heroku:22-cnb-build`][heroku-tags] | Available   |
 
-[`heroku/builder`][builder-tags] builder images feature Heroku's native Cloud Native Buildpacks. These buildpacks are optimized and make use of many CNB features. These builder images support Go, Java, Node.js, PHP, Python, Ruby, and Typescript codebases.
+[`heroku/builder`][builder-tags] builder images feature Heroku's native Cloud Native Buildpacks. These buildpacks are optimized and make use of many CNB features. These builder images support Go, Java (Maven, Gradle), Node.js, PHP, Python, Ruby, Scala and Typescript codebases.
```
[the remark this answers is not in the document]
Comment (Malax) [reply README.md:17]: This is very inconsistent. In current Heroku, Gadle is listed as a language which is IMO very incorrect. Also, Java on current Heroku means Maven which adds to the confusion. To make things worse, the actual language used by the app is indenpendent of the build tool. It's no problem to build Kotlin or Scala code with the Java buildpack. 

I proposed changing how we think about JVM languages and focus on the build tools instead. That means mentioning that we support Maven, Gradle, sbt and Leiningen on OpenJDK. Actual languages (Java, Kotlin, Scala, Clojure, Eta) would get their own landing pages to aid people coming to the platform, but documentation and buildpack names would be much closer to reality.

However, this is in the future and I don't know how the Node folks think about this issue since they have a similar problem. Ideally we would do it consistently in the future. 

Long story short: I chose the "Java (Maven, Gradle)" wording to continue to mention Gadle as we did so historically and people coming over will look for it when they are used to the way Heroku does it currently. At the same time, by grouping the build tools after Java, we no longer mis-label Gradle as a language and only imply Maven. This feels like the best way forward for now.

**Review by edmorley — APPROVED** [review]
